.github/workflows/sentryRelease.yml

```diff
@@ -0,0 +1,22 @@
+name: ReleaseWorkflow
+
+on:
+  release:
+    types: [published, prereleased]
+
+
+jobs:
+  createSentryRelease:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+      - name: Create a Sentry.io release
+        uses: tclindner/sentry-releases-action@v1.2.0
+        env:
+          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+          SENTRY_ORG: eigengraus-generator
+          SENTRY_PROJECT: eigengraus-generator
+        with:
+          tagName: ${{ github.ref }}
+          environment: qa
```

Comment:
```suggestion
  createRelease:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v1
        with:
          node-version: 12.x

      - name: Install dependencies
        run: yarn install
        run: yarn install-compiler

      - name: Build
        run: yarn build

      - name: Copy resources
        run: cp -r ./src/Resources ./gh-pages

      - name: Publish
         run: yarn gh-pages -d gh-pages

      - name: Create a Sentry.io release
        uses: tclindner/sentry-releases-action@v1.2.0
        env:
          SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
          SENTRY_ORG: eigengraus-generator
          SENTRY_PROJECT: eigengraus-generator
        with:
          tagName: ${{ github.ref }}
          environment: production
          sourceMapOptions: '{"include": ["gh-pages"]}'
```

Reply:
I'm fairly sure this will allow you to automate the entire deploy process. All you have to do is create a new tagged version on GitHub, and it should run all this automatically, I think.